public/faq.html

```diff
@@ -202,14 +202,14 @@ <h5 class="mb-0">
                                             style="text-transform: none;" data-mdb-toggle="collapse"
                                             data-mdb-target="#collapse8" aria-expanded="false"
                                             aria-controls="collapse8">
-                                            Q. Which wallets supports iDNA BEP20 ?
+                                            Q: Which wallets supports iDNA BEP-20 ?
                                         </button>
                                     </h5>
                                 </div>
                                 <div id="collapse8" class="collapse" aria-labelledby="heading8"
                                     data-parent="#faqExample">
                                     <div class="card-body">
-                                        MetaMask , TrustWallet ,SafePal And any wallet that supports BEP20 Tokens.
+                                        MetaMask , TrustWallet ,SafePal and any other wallet that supports BEP-20 Tokens.
```

Comment:
MetaMask, TrustWallet, SafePal and any other wallet that supports BEP-20 Tokens.